extensions/css-language-features/client/src/node/dropIntoEditor.ts

```diff
@@ -0,0 +1,89 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { Mimes, extractUriList } from './shared';
+
+class DropIntoEditorProvider implements vscode.DocumentDropEditProvider {
+	readonly id = 'relativePath';
+	readonly dropMimeTypes = ['text/uri-list'];
+
+	async provideDocumentDropEdits(
+		document: vscode.TextDocument,
+		position: vscode.Position,
+		dataTransfer: vscode.DataTransfer,
+		token: vscode.CancellationToken):
+		Promise<vscode.DocumentDropEdit | undefined> {
+
+		const enabled = vscode.workspace.getConfiguration('css', document).get('format.formatDroppedFiles', true);
+		if (!enabled) {
+			return;
+		}
+
+		if (token.isCancellationRequested) {
+			return;
+		}
+
+		return this._getUriListDropEdit(document, dataTransfer, position, token);
+	}
+
+	private async _getUriListDropEdit(document: vscode.TextDocument, dataTransfer: vscode.DataTransfer, position: vscode.Position, token: vscode.CancellationToken): Promise<vscode.DocumentDropEdit | undefined> {
+		if (token.isCancellationRequested) {
+			return undefined;
+		}
+
+		const urlList = await dataTransfer.get('text/uri-list')?.asString();
+		if (!urlList) {
+			return undefined;
+		}
+
+		const snippet = await extractUriList(document, urlList);
+		if (!snippet) {
+			return undefined;
+		}
+
+		const pasteAsFunction = checkDrop(document, position);
+
+		return {
+			id: this.id,
+			priority: this._getPriority(pasteAsFunction),
+			label: snippet.label,
+			insertText: snippet.snippet.value
+		};
+	}
+
+	private _getPriority(pasteAsFunction: boolean): number {
+		if (!pasteAsFunction) {
+			// Deprioritize in favor of default drop provider
+			return -10;
+		}
+		return 0;
+	}
+}
+
+// CHANGE to compare position to 'match' position
+function checkDrop(document: vscode.TextDocument, position: vscode.Position): boolean {
+	const regex = /\(".*"\)/g;
```

Comment:
Instead of checking the entire document, can you just check the current line? I believe that should be enough for css